include/exadg/solvers_and_preconditioners/newton/newton_solver.h

```diff
@@ -80,12 +80,11 @@ class Solver
       linear_operator.set_solution_linearization(solution);
 
       // determine whether to update the operator/preconditioner of the linearized problem
-      bool const threshold_exceeded = (newton_iterations % update.threshold_newton_iter == 0) ||
-                                      (linear_iterations_last > update.threshold_linear_iter);
+      bool const update_now =
+        update.do_update and (newton_iterations % update.update_every_newton_iter == 0);
```

Comment:
Why did you change the logic here?

Reply:
I think this feature (`threshold_linear_iter`) was never used, so I removed it.